app/assets/stylesheets/layout/_work_packages_full_view.sass

```diff
@@ -104,10 +104,13 @@ body.controller-work_packages.full-create
   border-right: 1px solid #ccc
   overflow-y: auto
   overflow-x: auto
-  width: 60%
+  flex: 2
+  position: relative
+  min-width: 360px
```

Comment:
Why is the min-width 360px and not any other number?

Reply:
Because this is the min-width of one column and therefore the smallest possible width of the left section so that this is still looking good.

Reply:
👍